src/Signal/Extra.elm

```diff
@@ -66,8 +70,30 @@ two!
 
 infixl 4 ~>
 
+{-| Apply a Signal of functions to another signal. Like `Task.andMap`, this
+provides a more attractive way to combine several signals together into a
```

Comment:
Let's not put in a value judgement like "attractive", when you can objectively say that this is more scalable<sup>†</sup> than `mapN`. 
<sup>†</sup> or more easily extensible, smaller diffs, something like that